README.md

```diff
@@ -1,57 +1,73 @@
+| `graph.jsx`       | Provides a helper function for calling MS Graph API.   |
+| `ui.jsx`          | Contains UI logic.                         |
+| `index.js`        | Contains the root component.               |
+| `.gitignore`      | Define what to ignore at commit time.      |
+| `CHANGELOG.md`    | List of changes to the sample.             |
+| `CONTRIBUTING.md` | Guidelines for contributing to the sample. |
+| `package.json`    | Package manifest for npm.                  |
+| `README.md`       | This README file.                          |
+| `LICENSE`         | The license for the sample.                |
+
+**Note:** This sample was bootstrapped using [Create React App](https://github.com/facebook/create-react-app).
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
+[Node.js](https://nodejs.org/en/) must be installed to run this sample.
 
-## Demo
+### Setup
 
-A demo app is included to show how to use the project.
+1. [Register a new application](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) in the [Azure Portal](https://portal.azure.com). Ensure that the application is enabled for the [authorization code flow with PKCE](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow). This will require that you redirect URI configured in the portal is of type `SPA`.
+1. Clone this repository `git clone https://github.com/Azure-Samples/ms-identity-javascript-react-spa.git`
+1. Open the [/src/authConfig.js](./src/authConfig.js) file and provide the required configuration values.
+1. On the command line, navigate to the root of the repository, and run `npm install` to install the project dependencies via npm.
 
-To run the demo, follow these steps:
+## Running the sample
 
-(Add steps to start up the demo)
+1. Configure authentication and authorization parameters:
+   1. Open `src/authConfig.js`
+   2. Replace the string `"Enter_the_Application_Id_Here"` with your app/client ID on AAD Portal.
+   3. Replace the string `"Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here"` with `"https://login.microsoftonline.com/common/"` (*note*: This is for multi-tenant applications located on the global Azure cloud. For more information, see the [documentation](https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-javascript-auth-code)).
+   4. Replace the string `"Enter_the_Redirect_Uri_Here"` with the redirect uri you setup on AAD Portal.
+2. Configure the parameters for calling MS Graph API:
+   2. Replace the string `"Enter_the_Graph_Endpoint_Herev1.0/me"` with `"https://graph.microsoft.com/v1.0/me"`.
```

Comment:
maybe we can add a note here?

`(*note*: This is for MS Graph instance located on the global Azure cloud. For more information, see the [documentation](https://docs.microsoft.com/en-us/graph/deployments))`

Reply:
Done